src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.cs

```diff
@@ -1284,30 +1284,7 @@ public override int GetHashCode()
         internal static unsafe int GetIgnoreCaseHash(string source)
```

Comment:
This wrapper should not be needed anymore. Can everybody just call the method on string?